tests/frontend/use-user.test.tsx

```diff
@@ -64,7 +66,7 @@ describe('context wrapper', () => {
   });
 
   test('should use a custom profileUrl', async () => {
-    const fetchSpy = jest.fn();
+    const fetchSpy = jest.fn().mockReturnValue({ then: () => Promise.resolve() });
```

Comment:
` const fetchSpy = jest.fn().mockResolvedValue({})`

I would always use native promises rather than try and create mocked thenables